core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/UserUniqueIDManger.java

```diff
@@ -224,6 +224,43 @@ public UniqueIDPaginatedSearchResult listUsers(PaginatedSearchResult paginatedSe
         return users;
     }
 
+    /**
+     * Get list of user's from array of user names.
+     * @param listUsers List of user names.
+     * @param userStoreManager User store manger instance.
+     * @return List of @see User objects.
+     */
+    public List<User> getListOfUsers(String[] listUsers, AbstractUserStoreManager userStoreManager) {
+
+        return getListOfUsers(Arrays.asList(listUsers), userStoreManager);
+    }
```

Comment:
Do we need this method? If required we can do the same conversion from the place we call the below method. WDYT?

Reply:
fixed with d041a0b227c594f7cd6f3e9e5877d60b4f06c00a